server/test/suites/categoryRouter.test.js

```diff
@@ -25,85 +25,87 @@ let testUser2;
 let testCategory1;
 let testCategory2;
 
-before(async () => {
-  // Connect to in-memory mongodb
-  mongoServer = new MongoMemoryServer();
-  const mongoUri = await mongoServer.getUri();
-  getMongoDB(mongoUri);
-
-  // Populate the in memory database
-  try {
-    testUser1 = new User({ email: 'lobster1@gmile.com' });
-    await testUser1.save();
-
-    testUser2 = new User({ email: 'lobster2@gmile.com' });
-    await testUser2.save();
+module.exports =  function categorySuite() {
+  before(async () => {
```

Comment:
is it not possible to move this common before() code to index.test.js? I know you mentioned this possibility offline, so I just wanted to check in.

Reply:
On second thought, I see that itemRouter tests will come in the next PR, so I think this can be optional here

Reply:
This will be changed. There's going to be a db set up for each test case, after discussing with Rica offline